Makefile

```diff
@@ -23,7 +23,7 @@ check:
 	PYTHONPATH=$(CURDIR) PYTHONDONTWRITEBYTECODE=1 python3 -m pytest --color=$(COLOR) --verbose --showlocals --cov=hardly --cov-report=$(COV_REPORT) $(TEST_TARGET)
 
 test-image: files/recipe-tests.yaml
-	$(CONTAINER_ENGINE) build --rm -t $(TEST_IMAGE) -f files/Containerfile.tests --build-arg SOURCE_BRANCH=$(SOURCE_BRANCH) .
+	$(CONTAINER_ENGINE) build --no-cache --rm -t $(TEST_IMAGE) -f files/Containerfile.tests --build-arg SOURCE_BRANCH=$(SOURCE_BRANCH) .
```

Comment:
This might not be something that is desired in every situation. If you think customization of these options is needed on a regular basis, then we should and a `BUILD_OPTIONS` variable, or something similar, which could be used in such cases.